NoSoft-SA/nspack
# Feature/temp tail

CHANGELOG entry:

### Added
- [Add AppConst::CR_FG.temp_tail_required_to_ship](https://github.com/NoSoft-SA/nspack/pull/457/commits/cbe66f40f3ee4bf7ab663314d880dc3d5bc868e1)
### Changed
- [Set requires_temp_tail to disabled when AppConst::CR_FG.temp_tail_required_to_ship](https://github.com/NoSoft-SA/nspack/pull/457/commits/d7f9576c994ff9514f7eb64bd94a8aa9e32d7a77) 


How to test this code via the UI:
Loads: create or edit load where client setting is set. 
temptail required should be greyed out
